src/scaffoldmaker/meshtypes/meshtype_3d_bladderurethra1.py

```diff
@@ -281,6 +412,30 @@ def getDefaultOptions(cls, parameterSetName='Default'):
             options['Urethra diameter 1'] = 0.75
             options['Urethra diameter 2'] = 0.65
             options['Urethra wall thickness'] = 0.25
+        if 'Mouse' in parameterSetName:
+            options['Major diameter'] = 30.0
+            options['Minor diameter'] = 30.0
+            options['Wall thickness'] = 0.2
+            options['Neck diameter 1'] = 5.0
+            options['Neck diameter 2'] = 7.0
+            options['Neck angle degrees'] = 25
+            options['Ureter position around'] = 0.67  # should be on the dorsal part (> 0.5)
+            options['Ureter position down'] = 0.83
+            options['Urethra diameter 1'] = 2.0
+            options['Urethra diameter 2'] = 2.0
+            options['Urethra wall thickness'] = 0.25
+        if 'Pig' in parameterSetName:
+            options['Major diameter'] = 30.0
+            options['Minor diameter'] = 30.0
+            options['Wall thickness'] = 0.3
+            options['Neck diameter 1'] = 8.0
+            options['Neck diameter 2'] = 7.5
+            options['Neck angle degrees'] = 25
+            options['Ureter position around'] = 0.67  # should be on the dorsal part (> 0.5)
+            options['Ureter position down'] = 0.83
+            options['Urethra diameter 1'] = 2.0
+            options['Urethra diameter 2'] = 2.0
+            options['Urethra wall thickness'] = 0.25
```

Comment:
Mouse, pig have followed cat in being quite inflated, rat is half inflated, human perhaps a little more. Need to consistently choose a state, probably partly inflated so regular shaped, not over inflated so strained, not under-inflated so wrinkled up, to be the generic reference state. Do a bit of research and propose what you think is the best state.